js-miniapp-bridge/src/common-bridge.ts

```diff
@@ -192,6 +192,38 @@ export class MiniAppBridge {
     });
   }
 
+  /**
+   * Associating loadBannerAd function to MiniAppBridge object.
+   * This function preloads Banner ad before they are requested for display.
+   * Can be called multiple times to pre-load multiple ads.
+   * @param {string} id ad unit id of the Banner ad that needs to be loaded.
+   */
+  loadBannerAd(id: string) {
+    return new Promise<string>((resolve, reject) => {
+      return this.executor.exec(
+        'loadAd',
+        { adType: AdTypes.BANNER, adUnitId: id },
+        loadSuccess => resolve(loadSuccess),
+        error => reject(error)
+      );
+    });
+  }
+
+  /**
+   * Associating showBannerAd function to MiniAppBridge object.
+   * @param {string} id ad unit id of the Banner ad that will be shown.
+   */
+  showBannerAd(id: string) {
```

Comment:
Unlike Interstitial & Reward ad, the Banner ad just only loads ad and it automatically show so you can omit this function and add more comment details on `loadBannerAd`.

Reply:
@minh-rakuten Oh, nice catch. I didn't realize that banner ads are displayed differently than interstitial/rewarded ads. Looks like you have to insert an `AdView` somewhere: https://developers.google.com/admob/android/banner#add_adview

But I guess mini app developers might still want to load their ads when the mini app first opens and then display them later. So could we programmatically create a hidden `AdView` when they call `loadBannerAd` and then it show it later when they call `showBannerAd`?

Reply:
Hi Guys.
Just to add in context, this will be living in a separate feature branch for some time. 
Native changes will need to come in to support this module.

Reply:
I'm not sure when we set `Visibility.GONE` that adview would still run normally 🤔.

Reply:
I did remove the loading part as suggested by Minh san! this section has only show banner ads.